### ISSUE TITLE
Tournament difficulty grav suit placement

### ISSUE THREAD

**Issue by Cortoth** [tewtal/itemrandomizerweb#44]
I got a tournament seed the other day where the gravity suit was at the Maridia reserve location.  I don't think this is a bug as nothing on the info page says it shouldn't happen.  I just think it was a bit too hard for tournament difficulty.  In my opinion, getting to the Maridia reserve spot without grav suit is harder than the rest of suit-less Maridia (up to botwoon energy) and even harder than lava dive.

I've only been running sm randos for a month so I know I'm still very new.  It's certainly possible there is some technique to suit-less maridia reserve that I don't know, but I just figured I'd put this out there just in case.

Thanks for all your work on this randomizer.  I hadn't touched SM in years and now I'm having a blast with it again.

**Comment by moriwizz** [comment]
I agree that it's a bad idea to put Gravity in unreachable Maridia locations.
